tests/securedrop/install.pm

```diff
@@ -0,0 +1,51 @@
+# The Qubes OS Project, https://www.qubes-os.org/
+#
+# Copyright (C) 2024 Francisco Rocha <francisco@freedom.press>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use base "installedtest";
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+
+    $self->select_gui_console;
+
+    assert_and_click("menu");
+    if (check_screen("menu-tab-favorites-active", 30)) {
+        # switch to apps tab
+        click_lastmatch();
+    }
+    assert_and_click("menu-vm-work");
+    assert_and_click("menu-vm-xterm");
```

Comment:
and it failed: https://openqa.qubes-os.org/tests/111752#step/install/6

You hit a trap :P `xterm` isn't in the menu by default, it's added in https://github.com/QubesOS/openqa-tests-qubesos/blob/main/tests/simple_gui_apps.pm. In tests that use xterm opened in a VM, I usually add a comment about it: https://github.com/QubesOS/openqa-tests-qubesos/blob/main/tests/screenlocker_idle_watch.pm#L22

Reply:
Xfce4-terminal should be fine. You'll need needle for the menu and then for the actual window (see any of the tests using xterm)